.rubocop.yml

```diff
@@ -81,6 +81,16 @@ Style/DotPosition:
 Style/DoubleNegation:
   Enabled: false
 
+Style/SymbolArray:
+  Enabled: false
+
+Style/IndentHeredoc:
```

Comment:
`IndentHeredoc` is introduced from Ruby 2.3.
However, we have to support Ruby 2.0.0.
So I disabled.

ref: https://github.com/bbatsov/rubocop/pull/4028